bootstrap.py

```diff
@@ -766,6 +767,40 @@ class MEOptions:
     disable_rhn_plugin()
 
 
+def enable_service(service, failonerror=True):
+    """
+    Helper function to enable a service using proper init system.
+    pass failonerror = False to make init system's commands non-fatal
+    """
+    if failonerror:
+        if os.path.exists("/run/systemd"):
+            exec_failexit("/usr/bin/systemctl enable %s" % (service))
+        else:
+            exec_failexit("/sbin/chkconfig %s on" % (service))
+    else:
+        if os.path.exists("/run/systemd"):
+            exec_failok("/usr/bin/systemctl enable %s" % (service))
+        else:
+            exec_failok("/sbin/chkconfig %s on" % (service))
+
+
+def start_service(service, command="start", failonerror=True):
```

Comment:
OK, I can change the function to allow any command and then avoid the default value of command (command = start) and keep the order suggested.